common/views/components/SearchFilters/SearchFiltersDesktop.tsx

```diff
@@ -13,62 +14,77 @@ import Icon from '../Icon/Icon';
 import DropdownButton from '@weco/common/views/components/DropdownButton/DropdownButton';
 import NumberInput from '@weco/common/views/components/NumberInput/NumberInput';
 import CheckboxRadio from '@weco/common/views/components/CheckboxRadio/CheckboxRadio';
-import dynamic from 'next/dynamic';
 import { SearchFiltersSharedProps } from '../SearchFilters/SearchFilters';
 import {
   CheckboxFilter as CheckboxFilterType,
   DateRangeFilter as DateRangeFilterType,
   ColorFilter as ColorFilterType,
   filterLabel,
+  Filter,
 } from '../../../services/catalogue/filters';
 import ModalMoreFilters from '../ModalMoreFilters/ModalMoreFilters';
 import { ResetActiveFilters } from './ResetActiveFilters';
 import ButtonSolid, { ButtonTypes } from '../ButtonSolid/ButtonSolid';
 import { filter } from '@weco/common/icons';
 import { themeValues } from '@weco/common/views/themes/config';
+import { useLayoutEffect } from 'react';
```

Comment:
We already have a `from react` import at the top of the document. I use an ESLint extension in VSCode that picks up whatever our config is and automatically makes changes on save, it's great for this sort of thing: //marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint